data/json/recipes/recipe_medsandchemicals.json

```diff
@@ -88,10 +88,8 @@
   "difficulty": 2,
   "time": 10000,
   "book_learn": [["adv_chemistry", 1] , ["textbook_chemistry", 1]],
-  "tools": [
-    [
-      [ "chemistry_set", 5 ]
-    ]
+  "qualities": [
+    {"id":"CHEM","level":1}
```

Comment:
I wanna know what sort of acid water IRL we can get from salt water to properly set `CHEM` quality.